schema/api.py

```
@@ -228,22 +238,34 @@
             joinall(greenlets, timeout=GREENLET_TIMEOUT)
 
-            cached_data = greenlets[0].value if greenlets[0].ready() else None
-            vulnerability = greenlets[1].value if greenlets[1].ready() else None
+            def safe_value(greenlet):
+                if not greenlet.ready():
+                    return None
+                if getattr(greenlet, "exception", None):
+                    return None
+                try:
+                    return greenlet.value
+                except Exception:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)